docs/roman/data_products/product_types.rst

```diff
@@ -60,26 +58,35 @@ the user is running the pipeline. The input for each optional step is the output
-| :ref:`sky_match <skymatch_step>`                  | asn             | skymatch (opt)               | ModelLibrary     | MJy/sr              | A list of _cal files                  |
-+---------------------------------------------------+-----------------+------------------------------+------------------+---------------------+---------------------------------------+
-| :ref:`outlier_detection <outlier_detection_step>` |                 | outlier_detection_step (opt) | ModelLibrary     | MJy/sr              | A list of _cal files                  |
-+---------------------------------------------------+-----------------+------------------------------+------------------+---------------------+---------------------------------------+
-| :ref:`resample <resample_step>`                   |                 | resamplestep (opt)           | ModelLibrary     | MJy/sr              | A list of _cal files                  |
-+---------------------------------------------------+-----------------+------------------------------+------------------+---------------------+---------------------------------------+
-|                                                   |                 | coadd                        | MosaicModel      | MJy/sr              | A 2D resampled image                  |
-+---------------------------------------------------+-----------------+------------------------------+------------------+---------------------+---------------------------------------+
++---------------------------------------------------+-----------------+------------------------------+---------------------+----------------------+---------------------------------------+
+| Pipeline Step                                     | Input           |  Output suffix               | Data Model          | Units                | Description                           |
++===================================================+=================+==============================+=====================+======================+=======================================+
+| :ref:`flux <flux_step>`                           | asn             | flux (opt)                   | ImageModel          | MJy/sr               | A 2D flux image                       |
++---------------------------------------------------+-----------------+------------------------------+---------------------+----------------------+---------------------------------------+
+| :ref:`sky_match <skymatch_step>`                  | asn             | skymatch (opt)               | ImageModel          | MJy/sr               | A 2D image with sky levels computed   |
++---------------------------------------------------+-----------------+------------------------------+---------------------+----------------------+---------------------------------------+
+| :ref:`outlier_detection <outlier_detection_step>` | asn             | outlier_detection_step (opt) | ImageModel          | MJy/sr               | A 2D image with outliers flagged      |
```

Comment:
In these steps ModelLibrary has been replaced by ImageModel.  For something like flux_step it can return either, but with an asn being sent in it seems to me that either a ModelLibrary, or maybe more explicitly a ModelLibrary of ImageModels would be better?  Meanwhile skymatch or outlier detection always do a model library.

Reply:
Updated